pkg/config/debugging.go

```diff
@@ -0,0 +1,25 @@
+package config
+
+type Debugging struct {
+	// Valid values are: "Normal", "Debug", "Trace", "TraceAll".
+	// Defaults to "Normal".
+	LogLevel string `json:"logLevel"`
+}
+
+// GetVerbosity returns the numerical value for LogLevel which is an enum
+func (c *Config) GetVerbosity() int {
+	var verbosity int
+	switch c.Debugging.LogLevel {
```

Comment:
What do you think about doing a `strings.ToLower(c.Debugging.LogLevel)` and match on lowercase to avoid the odd chance a user adds incorrect capitalization?

Reply:
That makes sense. This PR is already pretty big. Maybe we can do that in a follow-up?

Reply:
Yeah absolutely, I have no issue with it.

Reply:
I opened USHIFT-1043 to remember to do this later.